docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -159,6 +159,10 @@ property key | Description | Default |
 `plugin.kafka.topic_profilings` | Specify which Kafka topic name for Thread Profiling snapshot to report to. | `skywalking_profilings` |
 `plugin.kafka.topic_management` | Specify which Kafka topic name for the register or heartbeat data of Service Instance to report to. | `skywalking_managements` |
 `plugin.springannotation.classname_match_regex` |  Match spring beans with regular expression for the class name. Multiple expressions could be separated by a comma. This only works when `Spring annotation plugin` has been activated. | `All the spring beans tagged with @Bean,@Service,@Dao, or @Repository.` |
+`plugin.grpclog.service_host` | Specify which grpc server's host for log data to report to. | `127.0.0.1` |
+`plugin.grpclog.service_port` | Specify which grpc server's port for traces data to report to. | `8000` |
+`plugin.grpclog.max_message_size` | Specify the maximum size of log data for grpc client to report to. | `10MB` |
```

Comment:
The default value `10MB` misleads users that they can set it to something like `20MB`, please use `10485760`